README.md

```diff
@@ -129,7 +129,7 @@ $ dapr init --kubernetes
 ⌛  Making the jump to hyperspace...
 ℹ️  Note: this installation is recommended for testing purposes. For production environments, please use Helm
 
-✅  Deploying the Dapr Operator to your cluster...
+✅  Deploying the Dapr control plane to your cluster...
 ✅  Success! Dapr has been installed. To verify, run 'kubectl get pods -w' in your terminal
```

Comment:
based on the new CLI status command 
https://github.com/dapr/cli/issues/312
https://github.com/dapr/cli/pull/314

Can we also change this sentence

Success! Dapr has been installed. To verify, run 'kubectl get pods -w' in your terminal
to 
Success! Dapr has been installed. To verify, run 'kubectl get pods -w' or 'dapr status -k' in your terminal

Reply:
I have resolved it now.